src/EventParticipation.php

```diff
@@ -70,10 +70,11 @@ public static function translateStatus($status)
      */
     public function export()
     {
-        return [
-            'email' => $this->getUser()->getEmail(),
-            'responseStatus' => $this->getStatus(),
-        ];
+        $export = ['email' => $this->getUser()->getEmail()];
+        if (null !== $this->getStatus()) {
+            $export['responseStatus'] = $this->getStatus();
+        };
+
+        return $export;
```

Comment:
imo we should return the "undecided" status instead of not returning any statuses